cplusutilities/Download-generic.sh

```diff
@@ -112,7 +112,7 @@ get_paths()
     if [[ "$DATASET_WITH_CHILDREN" != "1" ]]
     then
         GENERIC_PATHS="$(make_generic_path $ALIEN_JDL_OUTPUTDIR)"
-        GENERIC_PATHS=("$GENERIC_PATHS/$trainname/*/$outputfile" )
+        GENERIC_PATHS=("$GENERIC_PATHS/$trainname/AOD/*/$outputfile" )
```

Comment:
@benedikt-voelkel @LuukVermunt  should we merge this change into master?

Reply:
Has this been approved?